src/handler/on-start/constant.ts

```diff
@@ -0,0 +1,9 @@
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+const NODE_ENV = process.env["NODE_ENV"] ?? "development";
+const production = NODE_ENV === "production";
+export const handlerRoot = production ? "dist" : "src";
+
+export const handlerFileFilter = production ? "*.js" : "*.ts";
```

Comment:
I don't think we need this file at this time, although it could be helpful in future, but could we remove them first?

Reply:
yea sure, when it comes to production, this will be required as it will not able to get those '.js' files in `dist/` folder, because its looking for the files from `src` folder. The reason why we have this issues is because i implemented the auto import files

Reply:
Yes, this make sense, but I didn't really use `tsc` to compile this file on production at this point. It is not a good practice on production, and I am willing to update it, but in another PR. I believe It will include a few updates on `package.json` as well before we can utilize `tsc`.

Reply:
didn't know we can compile for production without using `tsc`, I would like to know more about this

Reply:
It can, but bad practice and dangerous

Reply:
> didn't know we can compile for production without using `tsc`, I would like to know more about this

@pwkang I was using `npm run live` at production. Like @OysterD3 mentioned, it is a bad practice. But this PR is large enough and I will prefer do it in an other PR.